code/game/objects/items/radio/detpack.dm

```diff
@@ -255,7 +255,7 @@
 		set_frequency(signaler.frequency)
 		to_chat(user, "You transfer the frequency and code of [signaler] to [src].")
 		return
-	if(istype(target, /obj/item) || istype(target, /mob))
+	if(istype(target, /obj/item) || istype(target, /mob) || istype(target, /obj/vehicle/unmanned))
```

Comment:
whytho?

Reply:
i remembered this being a problem in the past for other unmanned and dont want it on this one especially, i can make a flag to put on this if you'd prefer